frontend/src/store/account/mutations.ts

```diff
@@ -1,11 +1,9 @@
 import Vue from 'vue'
-import { AccountStateT } from './types';
+import { AccountStateT, BalanceT } from './types';
 
 export const mutations = {
-  'RECEIVE_MY_BALANCE': function (state: AccountStateT, { balance, tokenName, demoPrintValue }: { balance: string, tokenName: string, demoPrintValue: string }): void {
```

Comment:
Did you run:
"npm run test"?

I think this breaks our tests

Reply:
Whoops. Sending you another PR in a second.

Reply:
Whoops. Fixed.